meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set name = "FlaGs" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chayan7/FlaGs/archive/v1.2.6.tar.gz
+  sha256: 0d64fa8a24ad1d79b9ea570549bf2278cd899ab15f65f63b5b65d7420a229f7d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install biopython==1.72 ete3 hmmer -vv"
+
+requirements:
+  build:
+  host:
+    - python =3.6
```

Comment:
You probably don't want to pin to a specific version of python. Unless you have a strong reason not to do it, I would use `python >=3.6` so that way anyone using 3.6, 3.7, 3.8 or 3.9 can install it. Same below in `run`.